libraries/lib-wave-track/WaveClip.h

```diff
@@ -184,6 +184,7 @@ class WAVE_TRACK_API WaveClip final :
 
    double GetPlayEndTime() const override;
 
+   // todo comment
    sampleCount GetPlayStartSample() const;
    sampleCount GetPlayEndSample() const;
```

Comment:
Should all mentions of "play" in function names or comments in this header file also change to "visible" for consistency?